platform/broadcom/sonic-platform-modules-inventec/d7054q28b/modules/transceiver.c

```diff
@@ -5,37 +5,152 @@
 #include "io_expander.h"
 #include "transceiver.h"
 
+/* For build single module using (Ex: ONL platform) */
+#include <linux/module.h>
```

Comment:
use optoe driver.

Reply:
optoe2 is used to get "eeprom" info. Updated inventec_d7054_util.py.
Inventec driver is used to get "lpmode", "presence" info that optoe does not support.